coraza.go

```diff
@@ -30,9 +31,9 @@ type corazaModule struct {
 	// deprecated
 	Include    []string `json:"include"`
 	Directives string   `json:"directives"`
-
-	logger *zap.Logger
-	waf    coraza.WAF
+	Tag        string   `json:"tag"`
```

Comment:
I wonder if tag is the right name. Is this something idiomatic in caddy?

Reply:
I use it to distinguish different vhosts, such as the domain  of the reverse proxy in the current site，what's your advice

Reply:
I think something like detaulfLabels or defaultTags being a key value would work.